libs/blocks/youtube/youtube.js

```diff
@@ -1,36 +1,111 @@
-import { createIntersectionObserver, isInTextNode } from '../../utils/utils.js';
+import { createIntersectionObserver, createTag, isInTextNode } from '../../utils/utils.js';
```

Comment:
Same questions about a source in this file, some things look a little different than how we'd usually do them or very specific (e.g. copied from a LiteYTEmbed example)

Maybe this comes from express though, and their coding style is a little different 🤔

Reply:
Part of the code is an optimized version of [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed) and [lite-vimeo-embed](https://github.com/luwes/lite-vimeo-embed), that's why it looks a bit different than how we usually do things.

Reply:
I wonder if we should link to the two projects within the files. It's not obvious from the get-go.

I'm personally fine with having the PR as history, but we could consider a comment with the source

Reply:
That's a good idea. I've added the links in my feedback commit 👍